src/base/BlockInterface.php

```diff
@@ -199,4 +206,11 @@ public function getIsContainer();
      * @return boolean
      */
     public function getIsDirtyDialogEnabled();
+
+    /**
+     * Whether preview is enabled for this block or not.
+     *
+     * @return boolean
+     */
+    public function getIsPreviewEnabled();
```

Comment:
@boehsermoe I think this is not need. If renderAdminPreview returns false, it states there is no preview image.